solidity/dashboard/src/services/slashed-tokens.service.js

```diff
@@ -78,6 +78,7 @@ const fetchSlashedTokens = async (web3Context) => {
       punishmentData = { amount, type: "SEIZED", event }
     }
 
+    if (!punishmentData.hasOwnProperty("amount")) continue
```

Comment:
I think the problem is in the `groupByTransactionHash` function. We missed the `amount` field in [`groupedByTransactionHash[transactionHash] = { ...returnValues }`](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/services/slashed-tokens.service.js#L114).

Should be `groupedByTransactionHash[transactionHash] = { ...returnValues, amount: returnValues.amount }`

Reply:
Fixed that place too. But in my case issue seems to be different: transaction '0x7f71b6dda7239fb70e0b97cf214176185b23c29162fb5f33cf6e8f5b644071e0' is neither in slashedTokensGroupedByTxtHash nor in seizedTokensGroupedByTxtHash - so results in empty punishmentData, which causes an error without extra check I initially added.

Reply:
Right! I see. But maybe let's just declare `let punishmentData` instead of `let punishmentData = {}` and in if statement
`if (!punishmentData || lte(punishmentData.amount, 0)) continue`

Reply:
sure, should be more clean / efficient